hack/install.sh

```diff
@@ -34,5 +34,7 @@ WITHOUT_PORT_FORWARD=true WITHOUT_VAULT=false make deploy
 # Configure the internal m4d endpoint in vault (the "secret" endpoint)
 WITHOUT_PORT_FORWARD=true make -C secret-provider configure-vault
```

Comment:
Why do we need a "secret" endpoint?
Manager takes care of the endpoint for dataset credentials, connector deployment configures the endpoint for external user credentials, gui configures the internal user credentials. What component uses "secret" endpoint?

Reply:
@tomersolomon1 ^

Reply:
This is not related to the PR